pkg/berglas/create.go

```diff
@@ -114,6 +118,31 @@ func (c *Client) secretManagerCreate(ctx context.Context, i *SecretManagerCreate
 		return nil, errors.New("missing plaintext")
 	}
 
+	var replication *secretspb.Replication
+	if i.Locations == nil {
+		replication = &secretspb.Replication{
+			Replication: &secretspb.Replication_Automatic_{
+				Automatic: &secretspb.Replication_Automatic{},
+			},
+		}
+	} else if len(i.Locations) > 0 {
```

Comment:
It feels weird to treat `nil` and `[]string` separately. Why not just treat `len(i.Locations) == 0` as the default automatic replication case?

Reply:
I've modified the code to treat both cases identically. I was just a tad worried it might hide some issues when someone incorrectly generates this list on-the-fly. Although I can't think of any reasonable use case where I would be doing so.